tensorboard/compat/tensorflow_stub/io/gfile.py

```diff
@@ -182,6 +182,7 @@ def isdir(self, dirname):
         return os.path.isdir(compat.as_bytes(dirname))
 
     def listdir(self, dirname):
+        import time; time.sleep(0.1) # simulate the network latency
```

Comment:
Is this for your development but not for the production?

Reply:
Hi, this is still work in progress. Please postpone the review. Thanks